adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -162,6 +162,13 @@ func CreateRoutesWithClusters(mgwSwagger model.MgwSwagger, upstreamCerts []byte,
 			logger.LoggerOasparser.Warnf("Production environment endpoints are not available for the resource %v:%v-%v",
 				apiTitle, apiVersion, resource.GetPath())
 		}
+		// Check whether the endpoint basepath contains a trailing slash.
+		if endpointBasepath != "" {
```

Comment:
you could use hasSuffix method here.
https://pkg.go.dev/strings

Reply:
Or rather TrimSuffix straightaway. Then that would be a single line code.
https://pkg.go.dev/strings#TrimSuffix